contracts/RMRK/utils/RMRKMintingUtils.sol

```diff
@@ -95,4 +110,13 @@ contract RMRKMintingUtils is OwnableLock {
     function _checkSaleIsOpen() private view {
         if (_totalSupply >= _maxSupply) revert RMRKMintOverMax();
     }
+
+    /**
+     * @notice Used to verify that that only the owner can mint tokens if lazy minting is not enabled.
+     * @dev In case lazy minting is not enable and msg sender is not the contract owner, the execution is reverted.
+     */
+    function _checkLazyMinting() internal view {
+        if (_lazyMintingEnabled == 0 && _msgSender() != owner())
```

Comment:
I probably went through this check 10 times just to be sure it doesn't bite us in the ass sometime in the future 😂